README.md

```diff
@@ -145,4 +145,75 @@ Updates to any environment variables must be updated here along side the core en
   - ansible-playbook configure_webservers.yml -i controlled_nodes.ini
 - Ensure that there are no errors
 - navigate to any host ip address in a public web browser:
-  - http://managednodeip:5000
\ No newline at end of file
+  - http://managednodeip:5000
+
+
+
+
+## Deploying on Server(s) using Docker
+
+### Notes for Corporate Proxy Configuration
```

Comment:
Nice work documenting this really clearly - I am going to pinch this and use it as my go-to resource for the next time I need to remember this one! 😄